test/source.js

```diff
@@ -67,11 +77,13 @@ function testSourceFiles(source) {
 
 function validateObjectsMember(topojson, format, fieldMap, t) {
   const fc = JSON.parse(topojson);
-  const fcPath = _.get(format, 'meta.feature_collection_path', 'data');
+  const fcPath = get(format, "meta.feature_collection_path", "data");
   const fieldsNames = fieldMap.map(f => f.name).sort();
+  //eslint-disable-next-line no-prototype-builtins
   t.ok(fc.objects.hasOwnProperty(fcPath));
```

Comment:
I think we can just [make lint happy](https://eslint.org/docs/latest/rules/no-prototype-builtins) by changing this.

```suggestion
  t.ok(Object.prototype.hasOwnProperty.call(fc.objects, fcPath));
```